pkg/restore/restore.go

```diff
@@ -456,6 +460,80 @@ func (ctx *context) execute() (Result, Result) {
 		}
 	}
 
+	// TODO: Re-order this logic so that CRs can be prioritized in the main loop, rather than after.
+
+	// Refresh and resolve based on CRDs added to the API server from the above restore loop.
+	// This is because CRDs have been added to the API groups but until we refresh, Velero doesn't know about the
+	// newly-added API groups in order to create the CRs from them.
+	if err := ctx.discoveryHelper.Refresh(); err != nil {
+		addVeleroError(&errs, errors.Wrap(err, "error refreshing discovery API"))
+	}
+	newResources, err := prioritizeResources(ctx.discoveryHelper, ctx.resourcePriorities, ctx.resourceIncludesExcludes, ctx.log)
+	if err != nil {
+		// Is this right?
+		addVeleroError(&errs, errors.Wrap(err, "error sorting resources"))
```

Comment:
similar comment to previous - I *think* the fallthrough should be fine because `newResources` would be nil here, so nothing new will be restored - right?

Can remove the L473 comment too.